### PR TITLE
Reserve Dockerfile has been created

### PR DESCRIPTION
**Pull request by vecinomio**
## Github Issue #

https://github.com/IYermakov/Snakes/issues/49

## What Are We Doing Here?

Reserve Dockerfile, which is based on original Tomcat image.

## How to Test and Verify

1. Check out this PR
2. To create image:  docker build -f webApp-based-on-Tomcat-Image/Dockerfile -t snakes:latest .
3. To run container locally: docker run -d -p 8080:8080 test-snakes

### Level

`Low`
### Required Testing

`Smoke` `Manual`

## Code Review Checklist

- [ ] Testing instructions are included. If not, make sure you leave us a line or two for the reason.

- [ ] Well documented, including updates to any necessary markdown files. When
we inevitably come back to this code it will only take hours to figure out, not
days.

- [ ] Make sure PR has only the changes required for the Jira issues.

- [ ] Have you make sure to follow the agreements in our process document?

- [ ] Have someone from different region than you review your changes. If not, make sure to leave a comment on why it is not required

- [ ] If it's applicable, document deployment plan for the changes.

### CODE REVIEW

File: webApp-based-on-Tomcat-Image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM tomcat:8
```
Comment (mike-unitskyi) [inline webApp-based-on-Tomcat-Image/Dockerfile]: any objections of using latest?

Comment (vecinomio) [reply webApp-based-on-Tomcat-Image/Dockerfile]: no problem

Comment (vecinomio) [reply webApp-based-on-Tomcat-Image/Dockerfile]: Done

**Comment by mike-unitskyi** [comment]
@vecinomio do you really need both?

**Pull request closed without merging**